config/environments/production.rb

```diff
@@ -33,6 +33,7 @@
     logger           = ActiveSupport::Logger.new(STDOUT)
     logger.formatter = config.log_formatter
     config.logger    = ActiveSupport::TaggedLogging.new(logger)
+    config.log_level = :debug
```

Comment:
I wouldn't recommend setting the log level in production to `debug`. This will cause lots of unnecessary output.